build/keys/sgx_mode_sim/.gitignore

```diff
@@ -1 +1 @@
-enclave_code_sign.pem
+enclave_code_sign.pem
```

Comment:
can you just revert this file. it looks like you removed the final newline in the file.

Reply:
thank you, done